docs/commercial/sharing.rst

```diff
@@ -34,6 +34,8 @@ Enabling
    
    You can always revoke access in the same panel.
 
+Users can log out by using the :ref:`Log Out <versions:Logging out>` button in the RTD flyout.
```

Comment:
We should eventually find a better way to describe the menu. I doubt folks know what we mean when core team says flyout.

For now, "link" is more clear, as we don't have any buttons in the flyout menu.

```suggestion
Users can log out by using the :ref:`Log Out <versions:Logging out>` link in the RTD flyout menu.
```